sui_core/src/consensus_adapter.rs

```diff
@@ -93,18 +89,12 @@ impl ConsensusAdapter {
     /// Attempt to reconnect with a the consensus node.
     async fn reconnect(
         address: SocketAddr,
-        buffer_size: usize,
     ) -> SuiResult<FramedWrite<TcpStream, LengthDelimitedCodec>> {
         let stream = TcpStream::connect(address)
             .await
             .map_err(|e| SuiError::ConsensusConnectionBroken(e.to_string()))?;
 
-        let stream = FramedWrite::new(
-            stream,
-            LengthDelimitedCodec::builder()
-                .max_frame_length(buffer_size)
-                .new_codec(),
-        );
+        let stream = FramedWrite::new(stream, LengthDelimitedCodec::builder().new_codec());
```

Comment:
Do we still need framed writes? Is not everything grpc across sui/narwhal?

Reply:
We still need to update the narwhal pointer in sui to a commit that includes the grpc changes we made. Once that's done we'll be able to drop the explicit framed streams.